cylc/flow/etc/cylc

```diff
@@ -177,25 +177,19 @@ if [[ -z "${CYLC_HOME}" ]]; then
     exit 1
 fi
 
-# Note "conda activate" fails to prepend the environment bin dir to PATH if
-# local jobs inherit the scheduler environment and bashrc has prepended other
-# paths here. We don't actually rely on PATH to find "cylc" below, but just in
-# case, this makes "conda activate" do the right thing:
-unset CONDA_SHLVL
-
-# If selecting a virtual environment, activate it
-if [[ -f "${CYLC_HOME}/bin/activate" ]]; then
-    # A Python venv or Conda pack installation
-    . "${CYLC_HOME}/bin/activate" || exit 1
-elif [[ -d "${CYLC_HOME}/conda-meta" && \
-        -f "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh" ]]; then
-    # A normal Conda environment
-    . "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh"
-    conda activate "${CYLC_HOME##*/}" || exit 1
-fi
+# Warn if executable not at $CYLC_HOME
 if [[ ! -x "${CYLC_HOME}/bin/${0##*/}" ]]; then
     echo 1>&2 "ERROR: ${0##*/} not found in ${CYLC_HOME}"
     exit 1
 fi
-# Execute the command in the selected installation.
+# Add CYLC_HOME to the start of the path to ensure the right version of python is used.
+PATH=${CYLC_HOME}/bin:${PATH}
```

Comment:
In most cases I don't think is needed (as long as you use the `--dest` option when using conda pack).
The only exception is `cylc hub` because `configurable-http-proxy` uses `#!/usr/bin/env node`.
Therefore  I suggest:
```
# Set PATH when running cylc hub so that configurable-http-proxy can find node
if [[ ${0##*/} =~ ^cylc && ${1:-} == "hub" && ! ${CYLC_VERSION:-} =~ ^7 ]]; then
  PATH=${CYLC_HOME}/bin:${PATH}
fi
```

Reply:
Not sure this is a good idea. It would work for running the Cylc executable due to the shebang in the Cylc Python script, however, if we don't modify the `$PATH` then calls we make out of Cylc will not pick up the Cylc environment.

Reply:
```suggestion
# Set PATH when running cylc hub so that configurable-http-proxy can find node
if [[ ${0##*/} =~ ^cylc && ${1:-} == "hub" && ! ${CYLC_VERSION:-} =~ ^7 ]]; then
  PATH=${CYLC_HOME}/bin:${PATH}
fi
```

Reply:
We've decided to go ahead with this change assuming we don't find find any issues during testing